R/crunchbox.R

```diff
@@ -41,6 +46,28 @@ crunchBox <- function (dataset, filters=crunch::filters(dataset), ...) {
     ## Add "where" after so that it no-ops if variablesFilter returns NULL (i.e. no filter)
     payload$where <- variablesFilter(dataset)
 
+    ## Add colors if they exist to the payload
+    if (!missing(brand_colors)) {
+        if (!is.list(brand_colors) ||
+            any(!names(brand_colors) %in% c("primary", "secondary", "message"))) {
+            halt(sQuote("brand_colors"), " must be a named list with only ",
+                 serialPaste(dQuote(c("primary", "secondary", "message")), collapse = "or"))
```

Comment:
Is there any way we can pass the colour scheme without a named list? For instance a character vector of length three which represented the three colours? I'm thinking of `ggplot2::scale_colour_manual()` as an example. Named lists can be a bit tricky for some users. Also a character vector would eliminate problems with  people naming the list incorrectly.

Reply:
👍